### ISSUE TITLE
Snippet editor headings

### ISSUE THREAD

**Issue by afercia** [Yoast/YoastSEO.js#362]
Heading hierarchy should be improved a bit, see https://github.com/Yoast/wordpress-seo/issues/4045
